### PR TITLE
added the context for the respective sections for EKS.

### PR DESCRIPTION
**Pull request by bainskb**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -350,7 +350,9 @@ The steps for deploying the model on your EKS cluster are described below.
     fmbench --config-file https://raw.githubusercontent.com/aws-samples/foundation-model-benchmarking-tool/main/src/fmbench/configs/llama3/8b/config-llama3-8b-eks-inf2.yml > fmbench.log 2>&1
     ```
 
-1. As the model is getting deployed you might want to run the following `kubectl` commands to monitor the deployment progress. Set the _model_namespace_ to `llama3` or `mistral` or a different model as appropriate.
+1. As the model is getting deployed you are able to track the progress using `kubectl` commands. Befor beign able to use this commands you have to make sure that your configuration file for kubectl is set apropriatly. if you have already installed the terraform script for DoEKS you can run the following comand to set the context for your future c=kubectl commands: `aws eks --region us-east-1 update-kubeconfig --name trainium-inferentia
```
Comment (aarora79) [inline README.md:353]: beign -> being
kubectl -> `kubectl`
"c=kubectl commands:" -> did not follow
`region` is hardcoded, we need to determine this through the cli `aws eks --region $(aws configure get region) update-kubeconfig --name trainium-inferentia`

File: misc/eks_cluster-creation_steps.md [hunk 2]
```diff
@@ -2,9 +2,11 @@
...
 
 
+1. The first thing that needs to be done is to make sure that you have enough inf2 on-demand vCPUs as most of the DoEKS blueprints utilize this specific instance. to increase service quota navigate to the service quota page for your respective region [service quota](https://us-east-1.console.aws.amazon.com/servicequotas/home?region=us-east-1). Then select services under the left side menu and search for "Amazon Elastic Compute Cloud (Amazon EC2)"". This will bring up the service quota page, here search for "inf" and there should be an option for "Running On-Demand Inf instances". Increase this quota to 300.
```
Comment (aarora79) [inline misc/eks_cluster-creation_steps.md:8]: Remove "The first thing that needs to be done is to make sure that you have" -> "Ensue that your account has enough `Inf2` on-demand VCPUs ..."
". to" -> ". To"
"respective region" -> "the region you are in"

**Review by aarora79 — CHANGES_REQUESTED** [review]
Please see comment inline and make the requested changes.

File: README.md (hunk 1 above)
Comment (antara678) [reply README.md:353]: made all these changes to other pull request

**Pull request closed without merging**